content/en/docs/intro/using_helm.md

```diff
@@ -59,6 +59,9 @@ The above searches for all `wordpress` charts on Artifact Hub.
 
 With no filter, `helm search hub` shows you all of the available charts.
 
+`helm search hub` exposes the repo URL which comes in handy when you are looking to add a new repo: `helm repo
+add [NAME] [URL]`.
+
```

Comment:
`helm search hub` exposes the URL to the location on artifacthub.io but not the actual Helm repo. On `helm search hub` you need to add the flag `--list-repo-url` to get that information. This flag needs to be documented to get the right URL to use with `helm repo add`